go.mod

```diff
@@ -3,23 +3,30 @@ module github.com/tendermint/tendermint
 go 1.15
 
 require (
+	github.com/Azure/go-ansiterm v0.0.0-20210617225240-d185dfc1b5a1 // indirect
```

Comment:
I think we don't have a lot of the indirect deps in the go mod, or haven't historically.

Reply:
I didn't add this myself—I guess it must have come from something `go get` or `go mod tidy` did. But I'll see if I can get rid of it without breaking things. 🙂

Reply:
what isn't? :p